src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java

```diff
@@ -392,29 +381,15 @@ public AboutToStartOrSubmitCallbackResponse sendEmailNotificationOnCaseWithdraw(
     private void sendWithdrawEmails(CaseData caseData, UserDetails userDetails, CaseDetails caseDetails) {
         if (PrlAppsConstants.C100_CASE_TYPE.equalsIgnoreCase(caseData.getCaseTypeOfApplication())) {
             solicitorEmailService.sendWithDrawEmailToSolicitorAfterIssuedState(caseDetails, userDetails);
-            Optional<List<Element<LocalCourtAdminEmail>>> localCourtAdmin = ofNullable(caseData.getLocalCourtAdmin());
-            if (localCourtAdmin.isPresent()) {
-                Optional<LocalCourtAdminEmail> localCourtAdminEmail = localCourtAdmin.get().stream().map(Element::getValue)
-                    .findFirst();
-                if (localCourtAdminEmail.isPresent()) {
-                    String email = localCourtAdminEmail.get().getEmail();
-                    log.info(" Sending notification to court email : {}", email);
-                    caseWorkerEmailService.sendWithdrawApplicationEmailToLocalCourt(caseDetails, email);
-                }
-            }
```

Comment:
why removed? this changes are added in 3.5 branch again. sorry confused.

Reply:
This has been removed in branch Release 4.0. While rebasing it got deleted from here too.